Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "parity-daemonize"
-version = "0.1.1"
+version = "0.1.2"
```

You've renamed the public error kind from `Windows` to `NotLinux`, is it not technically a breaking change?

---

Yeah, you are right did't think about that that. You want me change the major version correct?

---

Yeah, I think it should be 0.2
```suggestion
version = "0.2.0"
```